Baekjoon/cakejin/4949.py

```diff
@@ -0,0 +1,48 @@
+'''
+작성자: cakejin
+문제 링크:https://www.acmicpc.net/problem/4949
+카테고리: 자료 구조, 문자열, 스택
+문제해설: 균형잡힌 세상
+
+(),[]가 균형이루기 + 괄호 안 괄호도 균형
+주어진 문자열이 균형인지 판단
+
+스택에 ([나올때마다 쌓아두다가)]나오면 마지막에 쌓은 두개 빼기->끝까지 했을때 스택에 남아있는게 없으면 yes
+'''
+
+import sys
+
+while True:
+  
+  input_string = sys.stdin.readline().rstrip()
+  
+  if input_string == ".":
+    break
+  
+  a = len(input_string)
+  stack = []
+  balance = True
+
+
+  for i in range(a):
+    if input_string [i] == '(' or input_string[i] == '[':
```

Comment:
문자열 이터럴로 돌아도 될 것 같네요. `a = len(input_string)` 에서 길이를 저장할 필요도 없어 보입니다


```suggestion
  for s in input_string:
    if s == '(' or s == '[':
```

Reply:
수정했습니다~!